javascriptv3/example_code/bedrock-agent-runtime/actions/invoke-agent.js

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  BedrockAgentRuntimeClient,
+  InvokeAgentCommand,
+} from "@aws-sdk/client-bedrock-agent-runtime";
+
+/**
+ * @typedef {Object} ResponseBody
+ * @property {string} completion
+ */
+
+/**
+ * Invokes a Bedrock agent to run an inference using the input
+ * provided in the request body.
+ *
+ * @param {string} prompt - The prompt that you want the Agent to complete.
+ * @returns {string} The inference response (completion) from the agent.
```

Comment:
`sessionId` is missing from the JSDoc. Also, `@returns` is not needed as it can be inferred.